helix-term/src/ui/statusline.rs

```diff
@@ -421,10 +421,7 @@ where
         .map(|p| p.to_string_lossy())
         .unwrap_or_else(|| SCRATCH_BUFFER_NAME.into());
 
-    let title = format!(
-        " {} ",
-        path,
-    );
+    let title = format!(" {} ", path,);
```

Comment:
The formatter should have removed the trailing comma when it decided to make this a one liner.